apps/web/app/providers.tsx

```
@@ -25,24 +22,20 @@ if (item === 'true') {
-      </ChakraProvider>
-    </CacheProvider>
+    <RootProvider
+      theme={{
+        enabled: false,
+      }}
+    >
+      <PythonProvider>
+        <TooltipProvider>
+          <script
+            suppressHydrationWarning
+            dangerouslySetInnerHTML={{ __html: inject }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)